discovery-provider/plugins/notifications/src/__tests__/mappings/supportingRankUp.test.ts

```diff
@@ -68,4 +71,37 @@ describe('Supporting Rank Up Notification', () => {
     })
   })
 
+  test("Process email notification for supporting rank up", async () => {
+    await createUsers(processor.discoveryDB, [{ user_id: 1 }, { user_id: 2 }])
+    await createUserBankTx(processor.discoveryDB, [{ signature: '1', slot: 1 }])
+    await createSupporterRankUp(processor.discoveryDB, [{ sender_user_id: 2, receiver_user_id: 1, rank: 2 }])
+
+    const data: SupportingRankUpNotification = {
+      rank: 2,
+      sender_user_id: 2,
+      receiver_user_id: 1
+    }
```

Comment:
which one is the supporting rank up notif again? does it matter the sender and receiver are switched from the specifier and receiver in the notification declared below?

Reply:
i'll fix the test - it doesn't matter for this test

Reply:
will fix, supporting rank up will send to the receiver user_id - doesn't matter for test, bc we are just testing render.